extensions/sql-migration/src/wizard/createMigrationControllerDialog.ts

```diff
@@ -278,133 +265,115 @@ export class CreateMigrationControllerDialog {
 
 	private createControllerStatus(): azdata.FlexContainer {
 
-		const informationTextBox = this._view.modelBuilder.text().withProps({
-			value: constants.CONTROLLER_DIALOG_CONTROLLER_CONTAINER_DESCRIPTION
-		}).component();
-
-		const expressSetupTitle = this._view.modelBuilder.text().withProps({
-			value: constants.CONTROLLER_OPTION1_HEADING,
+		const setupIRHeadingText = this._view.modelBuilder.text().withProps({
+			value: constants.CONTROLLER_DIALOG_CONTROLLER_CONTAINER_HEADING,
 			CSSStyles: {
 				'font-weight': 'bold'
 			}
 		}).component();
 
-		const expressSetupLink = this._view.modelBuilder.hyperlink().withProps({
-			label: constants.CONTROLLER_OPTION1_SETUP_LINK_TEXT,
-			url: ''
+		const setupIRdescription = this._view.modelBuilder.text().withProps({
+			value: constants.CONTROLLER_DIALOG_CONTROLLER_CONTAINER_DESCRIPTION,
 		}).component();
 
-		expressSetupLink.onDidClick((e) => {
-			vscode.window.showInformationMessage(constants.FEATURE_NOT_AVAILABLE);
-		});
-
-		const manualSetupTitle = this._view.modelBuilder.text().withProps({
-			value: constants.CONTROLLER_OPTION2_HEADING,
-			CSSStyles: {
-				'font-weight': 'bold'
-			}
-		}).component();
-
-		const manualSetupButton = this._view.modelBuilder.hyperlink().withProps({
-			label: constants.CONTROLLER_OPTION2_STEP1,
-			url: 'https://www.microsoft.com/download/details.aspx?id=39717'
+		const irSetupStep1Text = this._view.modelBuilder.text().withProps({
+			value: constants.CONTROLLER_STEP1,
+			links: [
+				{
+					text: constants.CONTROLLER_STEP1_LINK,
+					url: 'https://www.microsoft.com/download/details.aspx?id=39717'
+				}
+			]
 		}).component();
 
-		const manualSetupSecondDescription = this._view.modelBuilder.text().withProps({
-			value: constants.CONTROLLER_OPTION2_STEP2
+		const irSetupStep2Text = this._view.modelBuilder.text().withProps({
+			value: constants.CONTROLLER_STEP2
 		}).component();
 
-		const connectionStatusTitle = this._view.modelBuilder.text().withProps({
-			value: constants.CONTROLLER_CONNECTION_STATUS,
+		const irSetupStep3Text = this._view.modelBuilder.hyperlink().withProps({
```

Comment:
I'd suggest adding a little spacing between the table and this link - they're a little too close I think